src/test/java/ua/softserveinc/tc/validator/RoomValidatorTest.java

```diff
@@ -0,0 +1,140 @@
+package ua.softserveinc.tc.validator;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+import ua.softserveinc.tc.constants.ValidationConstants;
+import ua.softserveinc.tc.dto.BookingDto;
+import ua.softserveinc.tc.dto.ChildDto;
+import ua.softserveinc.tc.dto.RoomDto;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * Created by Vitalij Fedyna on 25.12.16.
+ */
+public class RoomValidatorTest {
+
+  private RoomValidator roomValidator;
+
+  @Mock
+  private BookingDto bookingDto;
+
+  @Mock
+  private RoomDto roomDto;
+
+  @Mock
+  private Errors errors;
+
+  @Before
+  public void setRoomValidator() {
+    MockitoAnnotations.initMocks(this);
+    roomValidator = new RoomValidatorImpl();
+    roomDto = new RoomDto();
+    errors = new BindException(roomDto, "roomDto");
```

Comment:
I`m not sure that it is ok after annotation @mock create new object.  Because you never mock any methods from this object. I think you could delete annotation @Mock. But check it.